packages/@angular/cli/commands/test.ts

```diff
@@ -14,13 +14,15 @@ export interface TestOptions {
   build?: boolean;
   sourcemap?: boolean;
   progress?: boolean;
+  config: string;
 }
 
 
 const TestCommand = EmberTestCommand.extend({
   availableOptions: [
     { name: 'watch', type: Boolean, default: true, aliases: ['w'] },
     { name: 'code-coverage', type: Boolean, default: false, aliases: ['cc'] },
+    { name: 'config', type: String, aliases: ['c', 'cf'] },
```

Comment:
Updated with your comments. Though, 'cc' and 'c' for respectively code coverage and config is little bit scary, no ?